index.js

```diff
@@ -0,0 +1,141 @@
+const inputTitle = document.querySelector('.title');
+const inputAuthor = document.querySelector('.author');
+const addBtn = document.querySelector('.add-btn');
+const sectionBooks = document.querySelector('.books');
+const footer = document.getElementById('footer');
+
+const findHight = () => {
+  if (window.innerHeight > document.body.scrollHeight) {
+    footer.style.position = 'fixed';
+  } else {
+    footer.style.position = 'relative';
+  }
+};
+
+function loadBooks(bookCollection) {
+  sectionBooks.innerHTML = '';
+  /* eslint-disable */
```

Comment:
- [ OPTIONAL] Kindly over  eslint disable line and instead get used to fix the issue in the code manually